scipy/spatial/ckdtree/src/query.cxx

```diff
@@ -1,10 +1,5 @@
-/*
- * This would break SciPy with NumPy 1.6 so just accept the compiler
- * warning for now.
- * #define NPY_NO_DEPRECATED_API NPY_1_9_API_VERSION 
- *
- */
- 
+#define NPY_NO_DEPRECATED_API NPY_1_9_API_VERSION
```

Comment:
Me neither. I tried to switch to using `numpy_nodepr_api` in spatial/setup.py, but that only lead to more mysterious CI failures. Reverted it now.